app/src/main/res/layout/activity_main.xml

```diff
@@ -1,18 +1,75 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We can add a ScrollView here and also enable `adjustResize` in AndroidManifest.xml